Semicircle.js

```diff
@@ -89,6 +89,25 @@
                 !(startAngle === 0 && stopAngle > 359) &&
                 !(startAngle == stopAngle)
             );
+        },
+        _containsPoint: function (p) {
+            function normalize (angle) {
+                var newAngle = angle;
+                while (newAngle <= -Math.PI)
```

Comment:
Please adhere to the surrounding brace style: always use braces for `while`/`if` statements.